Extensions/norecommended.js

```diff
@@ -37,14 +37,11 @@ XKit.extensions.norecommended = new Object({
 
 		if (XKit.page.react) {
 			XKit.tools.add_css(`
-				.norecommended-note {
-					height: 1em;
-					color: var(--white-on-dark);
-					opacity: 0.4;
-					padding: var(--post-header-vertical-padding) var(--post-padding);
-				}
-				.norecommended-note ~ * {
-					display: none;
+				.norecommended-hidden,
+ 				.norecommended-hidden + :not(.norecommended-done) {
```

Comment:
i've not seen these spacer elements, but i assume they're not actually posts if they're never processed by norecommended? which means this would be clearer:

```suggestion
 				.norecommended-hidden + :not([data-id]) {
```